### DISCUSSION TITLE
🔗 Please add also a heading class for typography to kit (Related to #965 - AKA Preset Styles, Duplicate of #21325)

### DISCUSSION THREAD

**Discussion by brittaweller** [elementor/elementor#21212] [Feature Request]
### Prerequisites

- [ ] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In all our websites we use heading classes like .h1, .h2, .h3, .. to have control over the design of our headings.

A h2 may be the correct tag but we often want it to look like a h3 and so on.

### Describe the solution you'd like

Could you please just add the class as a selector to the file elementor/core/kits/documents/tabs/theme-style-typography.php (line 182..)

```
// Headings.
$this->add_element_controls( esc_html__( 'H1', 'elementor' ), 'h1', '{{WRAPPER}} h1, {{WRAPPER}} .h1' );
$this->add_element_controls( esc_html__( 'H2', 'elementor' ), 'h2', '{{WRAPPER}} h2, {{WRAPPER}} .h2' );
$this->add_element_controls( esc_html__( 'H3', 'elementor' ), 'h3', '{{WRAPPER}} h3, {{WRAPPER}} .h3' );
$this->add_element_controls( esc_html__( 'H4', 'elementor' ), 'h4', '{{WRAPPER}} h4, {{WRAPPER}} .h4' );
$this->add_element_controls( esc_html__( 'H5', 'elementor' ), 'h5', '{{WRAPPER}} h5, {{WRAPPER}} .h5' );
$this->add_element_controls( esc_html__( 'H6', 'elementor' ), 'h6', '{{WRAPPER}} h6, {{WRAPPER}} .h6' );
```

**Comment by nicholaszein** [comment]
Hi @brittaweller,

It seems you posted this request twice. Please, avoid duplicating Discussions, this makes it harder for other members to vote. 

I'm merging this request to #21325 which is on the same subject.
